tests-semanticdb/src/test/resources/metac.expect_2.13

```diff
@@ -126,8 +126,6 @@ local3 => param x: Int
 local4 => class $anon extends Object with SAM { +1 decls }
   Object => java/lang/Object#
   SAM => example/SingleAbstractMethod#SAM#
-local5 => param y: Int
```

Comment:
this is the one that started bothering me. it was here but not in occurrences (and you made a point before that a symbol without occurrences is useless).

Reply:
> and you made a point before that a symbol without occurrences is useless

Especially local ones, since we will never really find them or use them. Might be a bit different in thing like objects for case classes, but that should be at least in synthetics section